src/test/java/dk/sdu/se_f22/contentmodule/stopwords/SemesterProjekt1.4_stopWordFilter/src/StopWords.java

```diff
@@ -0,0 +1,161 @@
+import java.sql.*;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+import java.util.Scanner;
+
+public class StopWords implements IStopWords {
+    public static final String COLUMN_NAME ="name";
+    public static final String TABLE_Eng_WORDS ="words_to_removed";
+    public static final String WORDS_FROM_TOKEN_REMOVE="SELECT "+ COLUMN_NAME +" FROM "+ TABLE_Eng_WORDS +" WHERE "+
+            COLUMN_NAME +" = ?";
+    public static final String INSERT_DA_WORDS= "INSERT INTO "+ TABLE_Eng_WORDS +'('+COLUMN_NAME+")" +
+            " VALUES(?) ON CONFLICT DO NOTHING";
+    public static final String QUERY_DA_WORDS="SELECT * FROM " + TABLE_Eng_WORDS;
+    public static final String REMOVE_DA_WORDS="DELETE FROM " + TABLE_Eng_WORDS +" WHERE "+ COLUMN_NAME + " = ?";
+    private PreparedStatement queryWordsPrep;
+    private PreparedStatement deleteWordsPrep;
+    private PreparedStatement insertWordsPrep;
+    private PreparedStatement fromEngTokenPrep;
+    public static final String DB_NAME = "stop_words_Eng";
+    public static final String CONNECTION_STRING = "jdbc:postgresql://localhost:5432/" + DB_NAME;
+    public static final String DB_PASSWORD = "";
+    public static final String DB_USER = "postgres";
+    protected Statement statement ;
+    protected Connection connection ;
+    private static Scanner input = new Scanner(System.in);
+    private static List<String> tokenTester = new ArrayList<>();
+
+   public StopWords() {}
+
+//Note: the method below is defined for testing purposes, and not a part of the CMS-4
+    public static List<String> unfiltered() {
+        if (tokenTester.isEmpty()) {
+            System.out.println("Type a token ");
+            String token = input.nextLine();
+            String[] split = token.split(" ", 0);
+
+            for (String s : split) {
+                 s = s.replaceAll("\\s+","-").replaceAll("'", "");
+                tokenTester.add(s);
+            }
+        }
+        return tokenTester;
+    }

```

Comment:
If it's defined for testing purposes, perhaps it should be moved to a testing class.